actions/watch.ts

```diff
@@ -127,15 +131,49 @@ async function getTradeableOrderWithSignature(
   conditionalOrder: ConditionalOrder,
   contract: ComposableCoW
 ) {
+  const proof = conditionalOrder.proof ? conditionalOrder.proof.path : [];
+  const offchainInput = "0x";
+  contract.getTradeableOrderWithSignature;
+  const { to, data } =
+    await contract.populateTransaction.getTradeableOrderWithSignature(
+      owner,
+      conditionalOrder.params,
+      offchainInput,
+      proof
+    );
+
+  // await contract.provider
+  //   .estimateGas({
+  //     to: contract.address,
+  //     data,
+  //   })
+  //   .then((r) => {
+  //     console.log("[getTradeableOrderWithSignature:estimateGas] Success", r);
+  //   })
+  //   .catch((e) => {
+  //     console.error(
+  //       '[getTradeableOrderWithSignature:estimateGas] Error during "getTradeableOrderWithSignature" call: ',
+  //       e
+  //     );
+  //   });
```

Comment:
```suggestion
```

Reply:
will delete in a follow up PR. This was useful for some test, but i can remove it now